.travis.yml

```diff
@@ -30,9 +31,8 @@ before_install:
     else
       sudo mkdir -p /usr/local/man
       sudo chown -R "${USER}:admin" /usr/local/man
-      # brew update  # do not update to save build time
-      brew install python ccache hdf5 proj geos openmpi netcdf
-      brew uninstall gdal postgis numpy
+      HOMEBREW_NO_AUTO_UPDATE=1 brew install hdf5 proj geos open-mpi netcdf ccache
+      HOMEBREW_NO_AUTO_UPDATE=1 brew uninstall gdal postgis numpy  # WHY?
```

Comment:
I think maybe someone should answer this question before we merge 😅

Reply:
https://github.com/yt-project/yt/commit/6f6894e270bc8889509d7711a545c4ebbfd57b2e
I assume it was throwing some sort of error